android/app/src/main/java/com/mapeo/MainApplication.java

```diff
@@ -46,16 +46,12 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       @SuppressWarnings("UnnecessaryLocalVariable")
       List<ReactPackage> packages = new PackageList(this).getPackages();
-
-      // Packages that cannot be autolinked yet can be added manually here, for example:
+      // Packages that cannot be autolinked yet can be added manually here, for
+      // example:
       // packages.add(new MyReactNativePackage());
-
-      // Add unimodules
-      List<ReactPackage> unimodules = Arrays.<ReactPackage>asList(
-        new ModuleRegistryAdapter(mModuleRegistryProvider)
-      );
-      packages.addAll(unimodules);
-
+      packages.add(new ModuleRegistryAdapter(mModuleRegistryProvider));
```

Comment:
Does this actually work? My understanding was that this is a list of modules. These "unimodules" are the `expo-` modules that we use for:

- Camera
- Geolocation (GPS)
- Detecting device rotation for landscape photos

Can we check all of this still works in a build with this option?